debian/control

```diff
@@ -67,12 +67,14 @@ Depends: ${misc:Depends},
     wl-clipboard,
     xwayland
 Recommends:
-    sway-audio-idle-inhibit,
```

Comment:
So we need a changelog version sync to https://github.com/regolith-linux/SwayAudioIdleInhibit/blob/main/debian/changelog#L1.  Not clear to me if we are using a gh action to bump changelog or manually pushing commits.  Will check

Reply:
Assuming no additional actions required here. Merging PR so I can make a release